consensus/service/src/consensus_service.rs

```diff
@@ -20,7 +20,6 @@ use futures::executor::block_on;
 use grpcio::{EnvBuilder, Environment, Server, ServerBuilder};
 use mc_attest_api::attest_grpc::create_attested_api;
 use mc_attest_enclave_api::{ClientSession, PeerSession};
```

Comment:
The RaClient was the remote attestation client. DCAP uses the QE and PCE for completing the attestation locally.